packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx

```diff
@@ -342,24 +342,39 @@ const ManageLocationsForm = ({
     setSelectedLocations(locationOptions.map((l) => l.id))
   }
 
+  const locationsAvailable = !!locationOptions.length
+
   return (
     <div className="h-full w-full">
       <form onSubmit={handleSubmit}>
         <Modal.Content>
           <div>
-            <div className="border-grey-20 pb-base text-grey-50 flex w-full items-center justify-between border-b">
-              <div className="">
-                <p>Select locations that stock the selected variant</p>
-                <p>{`(${selectedLocations.length} of ${locationOptions.length} selected)`}</p>
+            <div
+              className={clsx(
+                "border-grey-20 text-grey-50 flex w-full items-center justify-between",
+                { "pb-base border-b": locationsAvailable }
+              )}
+            >
+              <div>
+                <p>
+                  {locationsAvailable
+                    ? "Select locations that stock the selected variant"
+                    : "You don't have any locations"}
```

Comment:
suggestion: singular? wdyt?

Reply:
As above, the plural is better copy imo, as you can have multiple. You'd say "You do not have a stock location" if you could only ever have one.

Reply:
For example would you « you don’t have any car » or « you don’t have any cars »? If that make sense

Reply:
I am also in favor of the plural version